src/controllers/userController.js

```diff
@@ -1,29 +1,33 @@
-const userService = require('../services/userService');
-const logger = require('../helpers/logger');
+const userService = require('../services/userService')
+const logger = require('../helpers/logger')
 
 const createUser = async (req, res) => {
-    const { email } = req.body;
+    const { email } = req.body
 
-    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
     if (!email || !emailRegex.test(email)) {
         logger.error(`Invalid email ${email} address`)
-        return res.status(400).json({ message: `Invalid email address` });
+        return res.status(400).json({ message: `Invalid email address` })
     }
 
     try {
-        const result = await userService.createUser(email);
+        const result = await userService.createUser(email)
         if (result) {
-            logger.info(`Email ${email} was added to the database`);
-            res.status(200).json({ message: `New email was added to the database` });
+            logger.info(`Email ${email} was added to the database`)
+            res.status(200).json({
```

Comment:
you can remove else statement with return res... from if block, but this is a matter of your taste